docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
   backend:
     build: ./backend
-    command: npm run start:dev
+    command: sh -c "npm install && npm run start:dev"
```

Comment:
Is it necessary to run `npm install` every time we start the backend service? I'd say running `docker-compose build` when we introduce new dependencies to the project is quite OK.

Reply:
Good spot, will remove.